cfgov/unprocessed/css/molecules/global-eyebrow.less

```diff
@@ -0,0 +1,105 @@
+/* topdoc
+  name: Social Media
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-social-media">
+            <h5 class="m-social-media_heading">Share this</h5>
+            <ul class="list">
+                <li class="list_item">
+                    <a class="m-social-media_icon" href="#">
+                        <span class="cf-icon cf-icon-email-social-square"></span>
+                        <span class="u-visually-hidden">
+                            Share by email
+                        </span>
+                    </a>
+                </li>
+            </ul>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-social-media
+            .m-social-media_heading
+            ul.list
+              li.list_item
+                .m-social-media_icon
+                  span.cf-icon
+                  span.u-visually-hidden
+  tags:
+    - cfgov-molecules
+*/
+
+.m-global-eyebrow {
+    .webfont-regular();
+
+    font-size: unit( 12px / @base-font-size-px, em);
+
+    &_tagline {
+        padding-left: 37px; //22px flag + 15px spacing
+        background: url('../img/us-flag_22x13.png') no-repeat 0 0;
+
+        .respond-to-dpi(2, {
+            background-image: url('../img/us-flag_44x26.png');
+            background-size: 22px;
+        });
+
+        &-usa {
+            white-space: nowrap;
+        }
+    }
+
+    &_languages {
+        .list__horizontal();
+        .list__unstyled();
```

Comment:
These are using properties of an unstyled horizontal list (horizontal layout, no bullets), but are using different padding, so it seems misleading to have it in the markup when it differs from what the CF docs show for unstyled, horizontal lists.